nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenType.scala

```diff
@@ -58,7 +58,7 @@ trait NirGenType { self: NirGenPhase =>
   def genArrayCode(st: SimpleType): Char =
     genPrimCode(st.targs.head)
 
-  def genBoxType(st: SimpleType): nir.Type = st.sym match {
+  def genBoxType(st: SimpleType): scala.scalanative.nir.Type = st.sym match {
```

Comment:
Why are all these additional `scala.scalanative.` required, in this file?

Reply:
Woah, seems that IntelliJ decided to "optimise imports" when I refactored.